### PR TITLE
Restore gravity compensation and native KUKA position control after merge

### PR DESCRIPTION
**Pull request by marcoesposito1988**

### CODE REVIEW

File: lwr_hw/krl/ros_position.src [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+bas(#tool,1)
+;input correct tool number here and restart
+
+;stay at the same position
+ptp $pos_act_mes
+;HALT
+
+; Joint position control
+$stiffness.strategy = 10
+$stiffness.axisstiffness={ A1 1000, A2 1000, A3 1000, A4 1000, A5 1000, A6 1000, E1 1000}
```
Comment (carlosjoserg) [inline lwr_hw/krl/ros_position.src]: If this is joint position control only, you don't need this. And neither the next ones until the `$stiffness.commit=true`.

**Comment by carlosjoserg** [comment]
Only that fix in the script. I still would like to have one single script, but in the meantime we can work with this as well. 

I should change the `ros_control.src` -> `ros_impedance.src` for consistency then.

**Comment by marcoesposito1988** [comment]
Done! I am not in the lab at the moment, I will test it asap.

**Comment by marcoesposito1988** [comment]
Actually after testing your master branch I am reconsidering if the position interface is at all necessary. 

Before merging multi-robot-test I just couldn't use impedance mode because of all kinds of errors coming from the KUKA box, but now it works flawlessly. I guess this is because you are not sending the effort coming from the PID anymore, but you are using the internal KUKA controller and just sending it the target position, am I right? Are you still using any custom controller with the arm?

**Comment by carlosjoserg** [comment]
The position interface is still needed, specially to avoid the PID tuning, which is already done by KUKA in that interface. I would leave the effort interface for people that need that for other apps, like bilateral teleoperation, etc.

> I guess this is because you are not sending the effort coming from the ROS PID controller anymore,

Exactly. I'm using __tau__ = __K__ __q_FRI__ ( + D(q)<-0 + tau_FRI<-0 + __f_dyn()__ ). 

This is what I call `StiffnessJointInterface` in #19. Right know you can control both K and q_FRI with slides. You can also fix the K, and use it as a position-controlled robot. However, it is not as precise as using the native position control strategy, that's why it is still needed. In fact, it acts as a P-controller, and K is the proportional gaing, so integral and derivative terms are missing.

What I don't like so far is having to switch scripts in the robot (which implies that you need to restart FRI), and for which I'd like to have a single KRL script.

**Pull request merged**